modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -24,10 +24,14 @@ When you hover over the `i` to the right of each rule, the rule ID range for tha
 +
 image::waf-rule-set.png[]
 +
-. In *Advanced performance options*, you can select *Disable body scanner*. If you do not check this option, the request body is scanned.
+. In *Advanced performance options*, select from the following options: +
+* *Disable body scanner*. If you do not check this option, the request body is scanned.
 +
 [NOTE]
-The *Disable body scanner* option must be checked if any valid responses will exceed 512 KB. If request body checking is enabled, the HTTP Limits policy “Maximum Message Size” should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting the resources.
+If a body is larger than 1 MB, the scan is skipped, even if you do not select *Disable body scanner*. If request body checking is enabled (default), the HTTP Limits policy *Maximum Message Size* should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting resources.
+* *Detect sensitive information* - You can select this option only if the *Disable body scanner* option is not selected. +
+If you select this option, the request body is scanned for sensitive information, and when you apply the WAF policy to Runtime Fabric, the log summary (`rtfWafSecurityPolicySummary`) records the count and number (rule ID) of the sensitive information pattern that is detected. +
+An `rtfSidSummary` log is also generated, which contains the API, rule, direction, and count for instances of sensitive information.
```

Comment:
"which contains the API, rule, direction, and count"...
please remove API from this sentence, for 4/20 release, there's no **API name** in the log. @JaiHarpalani can confirm.
So the sentence should be: "..., which contains the rule, direction, and count for instances of sensitive information"